docs/modules/ROOT/pages/utils/_class_hashes.adoc

```diff
@@ -0,0 +1,10 @@
+// Version
+:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.3.1[cairo 2.3.1]
+
+// Class Hashes
+:account-class-hash: 0x016c6081eb34ad1e0c5513234ed0c025b3c7f305902d291bad534cd6474c85bc
+:erc20-class-hash: 0x01d7085cee580ba542cdb5709bda80ebfe8bdede664261a3e6af92994d9a1de7
+:erc721-class-hash: 0x04376782cbcd20a05f8e742e96150757383dab737ab3e791b8505ad856756907
+
+// Presets page
+:class-hashes-page: xref:presets.adoc[Compiled class hash]
```

Comment:
```suggestion
:presets-page: xref:presets.adoc[Compiled class hash]
```
not even sure it's a better change, but that's how the section is actually called and it might make more sense to anyone reading the code (not generated docs)